src/hotspot/os_cpu/linux_aarch64/os_linux_aarch64.cpp

```diff
@@ -433,18 +433,23 @@ extern "C" {
         *(to--) = *(from--);
     }
   }
+
+  inline void atomic_copy64(const volatile void *src, volatile void *dst) {
+    *(jlong *) dst = *(const jlong *) src;
+  }
+
   void _Copy_conjoint_jlongs_atomic(const jlong* from, jlong* to, size_t count) {
     if (from > to) {
       const jlong *end = from + count;
       while (from < end)
-        os::atomic_copy64(from++, to++);
+        atomic_copy64(from++, to++);
```

Comment:
If `atomic_copy64` was a shared API with platform-specific implementations we could eliminate all these copies of `_Copy_conjoint_jlongs_atomic`.